stories/example-screen-staff-list.stories.js

```diff
@@ -289,322 +289,105 @@ const tabsDoubleRowHTML = `
       </a>
     </li>
   </ul>
-  <ul class="nav nav-tabs bg-white border border-light border-top-0 ps-7 pe-7">
-    <li class="nav-item">
-      <a class="nav-link active fs-5" href="#">
+
+  <ul class="nav nav-tabs bg-white ps-7 pe-7">
+    <li class="nav-item py-2 px-3">
+      <a class="fs-6 text-primary" href="#" style="font-weight: 400;">
```

Comment:
I realize the mockup has the subtab row being smaller font-size than the primary tabs, but we have decided against that and want to use the same `fs-5` font-size for both rows, please fix all `fs-6` to `fs-5`.